x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/jwt/JwtRealmTests.java

```diff
@@ -717,7 +717,7 @@ private void multipleRealmsAuthenticateJwtHelper(
                 assertThat(new TreeSet<>(Arrays.asList(user.roles())), equalTo(new TreeSet<>(Arrays.asList(authenticatedUser.roles()))));
                 if (jwtIssuerAndRealm.realm.delegatedAuthorizationSupport.hasDelegation()) {
                     assertThat(user.metadata(), is(equalTo(authenticatedUser.metadata()))); // delegated authz returns user's
-                                                                                            // metadata
+                    // metadata
```

Comment:
Err can we revert this change. The format looks off to me. Or make the trailing comment into its own line.

Reply:
Resolved. I have no idea what that comment wrapped to the next line. Fixed it.